xml/OptionPanel.xml

```diff
@@ -20,7 +20,7 @@
     outline="grey"
     showAnimation="SlideIn_Right"
     hideAnimation="SlideOut_Right"
-    animationDuration="0.1" />
+    animationDuration="0.2" />
```

Comment:
Unrelated change, is this intentional?

Reply:
Against the principle, but yeah this was intended. Upon testing with the OptionPanel it felt a bit too quick on the animation.